src/comments/comments.service.ts

```diff
@@ -0,0 +1,115 @@
+import {
+  ConflictException,
+  forwardRef,
+  Inject,
+  Injectable,
+  NotFoundException,
+} from '@nestjs/common';
+import { CreateCommentDto } from './dto/create-comment.dto';
+import { UpdateCommentDto } from './dto/update-comment.dto';
+import { PrismaService } from 'src/prisma/prisma.service';
+import { UsersService } from 'src/users/users.service';
+
+@Injectable()
+export class CommentsService {
+  constructor(
+    private readonly prisma: PrismaService,
+    @Inject(forwardRef(() => UsersService))
+    private userService: UsersService,
+  ) {}
+
+  async create(
+    uuid: string,
+    password: string,
+    postId: string,
+    createCommentDto: CreateCommentDto,
+  ) {
+    await this.userService.getUser(uuid, password);
+    const result = await this.prisma.comments.create({
+      data: {
+        contents: createCommentDto.contents,
+        postId: postId,
+        userId: uuid,
+      },
+    });
+    return result;
+  }
+
+  async findCommentList(postId: string) {
+    const commentList = await this.prisma.comments.findMany({
+      where: { postId: postId, deletedAt: null },
+      orderBy: { createdAt: 'desc' },
+      include: { author: true },
+    });
+    return {
+      commentList: commentList.map((comment) => {
+        return {
+          uuid: comment.userId,
+          id: comment.author.id,
+          contents: comment.contents,
+        };
+      }),
+    };
```

Comment:
```suggestion
    return commentList.map((comment) => {
        return {
          uuid: comment.userId,
          id: comment.author.id,
          contents: comment.contents,
        };
      });
```
이렇게 내보내는 필드값이 하나인 경우에 굳이 새로운 object 만들어서 리턴하기 보다는, 그냥 리턴해주고 컨트롤러 쪽에서 아래 코드처럼 처리하는 게 보기 깔끔할 수 있어요.
```js
//controller
...
const commentList = this.commentsService.findCommentList(postId);
return { commentList }
```